src/api/admin_user/handlers/update_admin_user_handler.rs

```diff
@@ -124,5 +124,10 @@ pub async fn update_admin_user_handler(
         .await?;
 
     println!("{admin_user_model:?}");
+    
+    session
+        .insert("success_message", "Admin User edited successfully!")
+        .expect("Could not store the validation errors into session.");
```

Comment:
please fixed this error message. 

`An error occurred while adding success_message during update admin user.`